content/concepts/transports/quic.md

```diff
@@ -95,8 +95,30 @@ IDs are authenticated in the
 
 {{< alert icon="💡" context="note" text="To be clear, there is no additional security handshake and stream muxer needed as QUIC provides all of this by default. This also means that establishing a libp2p connection between two nodes using QUIC only takes a single RTT." />}}
 
-Following the multiaddress format described earlier, a standard QUIC connection will
-look like: `/ip4/127.0.0.1/udp/65432/quic/`.
+Following the multiaddress format, a standard QUIC connection will
+look like: `/ip4/127.0.0.1/udp/65432/quicv1/`.
+
+### QUIC v0
+
+The initial implementation of QUIC for some implementations (e.g. go-libp2p),
+"QUIC v0", was based on
+[draft-ietf-quic-transport-29](https://datatracker.ietf.org/doc/html/draft-ietf-quic-transport-29)
+(or simply, **draft-29**) by the IETF as [RFC 9000](https://datatracker.ietf.org/doc/html/rfc9000)
+was yet to be deployed. Many nodes still use draft-29 as their deployed version. A codepoint, `quicv1`,
+is used to distinguish between the two versions.
+
+A standard QUIC connection will look like: `/ip4/127.0.0.1/udp/65432/quicv1/`, whereas the initial
+codepoint, `quic` (e.g. `/ip4/127.0.0.1/udp/65432/quic/`) defaults to draft-29.
+
+Nodes that support multiple versions can offer them on the same port.
+Since QUIC packets contain the version number and any (multi-version)
+QUIC stack will be able to handle packets from different QUIC versions,
+demultiplexing works as intended.
+
+Due to version negotiation, a 1 RTT penalty will incur when connecting a QUIC v1 node to a
+legacy node.
```

Comment:
That _would_ be the case if we didn't have the quic-v1 code point. But since we now have the new code point, we _know_ the version in advance, so we don't waste 1 RTT on version negotiation.